pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -883,10 +876,7 @@ def tune_model(
         custom_grid: Optional[Union[Dict[str, list], Any]] = None,
         optimize: str = "smape",
         custom_scorer=None,
-        search_library: str = "pycaret",
```

Comment:
I wonder if it's a good idea to remove this. We will want to add bayesian search at some point. Perhaps it would be better to keep `search_library` but only allow `pycaret` as a value?

Reply:
We can add it back later when it is needed. Right now it is redundant.